src/users/userSelectors.js

```diff
@@ -159,13 +159,14 @@ export const getActiveUsersByEmail: Selector<Map<string, UserOrBot>> = createSel
 export const getAccountDetailsUserForEmail: Selector<UserOrBot, string> = createSelector(
   (state, email) => email,
   state => getAllUsersByEmail(state),
-  (email, allUsersByEmail) => {
+  (state, email) => getActiveUsersByEmail(state).has(email),
+  (email, allUsersByEmail, is_active) => {
```

Comment:
Style nit: prefer `camelCase` to `snake_case` for anything that isn't being serialized to the network.